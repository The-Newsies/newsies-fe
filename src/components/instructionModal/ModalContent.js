import React from 'react';
import PropTypes from 'prop-types';

export default function ModalContent({ isClosed, userName }) {
  return (
    <div>
      <button onClick={isClosed}>Got it X</button>
      <h2>Hello {userName}</h2>
      <section>
        <p>Will need to write a description here about what the collections
          page is for and how to navigate the trending and search pages to create collections.
        </p>
      </section>
    </div>
  );
}

ModalContent.propTypes = {
  isClosed: PropTypes.func.isRequired,
  userName: PropTypes.string.isRequired
};

