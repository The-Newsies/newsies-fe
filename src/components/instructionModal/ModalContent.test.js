import React from 'react';
import { shallow } from 'enzyme';
import ModalContent from './ModalContent';

describe('ModalContent component', () => {
  it('renders ModalContent', () => {
    const wrapper = shallow(<ModalContent isClosed={()=> {}} userName='Carlos' />);
    expect(wrapper).toMatchSnapshot();
  });
});
