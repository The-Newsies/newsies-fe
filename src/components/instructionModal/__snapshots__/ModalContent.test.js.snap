// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalContent component renders ModalContent 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    Got it X
  </button>
  <h2>
    Hello 
    Carlos
  </h2>
  <section>
    <p>
      Will need to write a description here about what the collections page is for and how to navigate the trending and search pages to create collections.
    </p>
  </section>
</div>
`;
